scripts/build.mjs

```diff
@@ -0,0 +1,84 @@
+import esbuild from "esbuild";
+import { copyFile, readFile, writeFile, rm } from "fs/promises";
+import { glob } from "glob";
+
+const sharedOptions = {
+  sourcemap: true,
```

Comment:
I'm seeing noticeable differences in the generated sourcemaps from one version and the other. The 2 main differences I see are: 

- `Pika`'s generated sourcemaps are pointing to `dist-src/` and the ones generated by ESBuild are pointing to `src/`. I guess we don't want this so we need the `sourcemaps: "external"` option. Does this make sense?
- `Pika`'s generated sourcemaps files are not including `sourcesContent` but the ones generated by ESBuild yes. I guess we don't want to emit that by using `sourcesContent: false`. What do you think?

```suggestion
  sourcemap: external,
  sourcesContent: false
```

Reply:
Wouldn't it be better to have them point to the source TS files instead of `dist-src/`?

That way the files can stay independent from each other

Reply:
Let's just make it output the same as Pika for now, and we can revisit these points later